### PR TITLE
Describe breaking change: KeycloakService constructor

### PR DESCRIPTION
**Pull request by Klaas-Ritense**

### CODE REVIEW

File: release-notes/major9/09.20.0/migration.md [hunk 1]
```diff
@@ -2,20 +2,17 @@
...
-
-      Description
-  2. **Step2**
-
-      Description
+* **Added extra parameter to KeycloakService**
+
+  If your project has a custom implementation of the KeycloakService, your project might break due to a missing
+  parameter in the constructor called `keycloakClientName`. The solution in simple: Just add an empty string for
+  the `keycloakClientName` parameter. Or, a more elegant solution would look as followed:
```
Comment (mritense) [inline release-notes/major9/09.20.0/migration.md:9]: as follows

Comment (Klaas-Ritense) [reply release-notes/major9/09.20.0/migration.md]: fixed

File: release-notes/major9/09.20.0/migration.md [hunk 2]
```diff
@@ -2,20 +2,17 @@
...
-  1. **Step1**
-
-      Description
-  2. **Step2**
-
-      Description
+* **Added extra parameter to KeycloakService**
+
+  If your project has a custom implementation of the KeycloakService, your project might break due to a missing
+  parameter in the constructor called `keycloakClientName`. The solution in simple: Just add an empty string for
```
Comment (marijnritense) [inline release-notes/major9/09.20.0/migration.md:8]: This text feels very subjective. What's elegant and what isn't is up to debate. Maybe rephrase this entire section to something like:

* ** Added name parameter to KeycloakService**

  Custom implementations of the KeycloakService now require `keycloakClientName` as a parameter in the super constructor. Either set this value to an empty string (e.g. `super(properties, "")` or do the following:

  ```java
  public CustomKeycloakService(
      KeycloakSpringBootProperties properties,
      @Value("${valtimo.keycloak.client:}") String keycloakClientName
  ) {
      super(properties, keycloakClientName);
      this.properties = properties;
  }
  ```

Comment (Klaas-Ritense) [reply release-notes/major9/09.20.0/migration.md]: changed

**Review by Rick-Ritense — APPROVED** [review]
